src/components/NanoContract/NanoContractDetails.component.js

```diff
@@ -43,7 +43,7 @@ const getNanoContractDetails = (ncId) => (state) => {
    * quickly, therefore the user will not have time to see the default state.
```

Comment:
Are you going to rename the component (to remove `.component.` in this PR?

Reply:
No. I can do it in another PR.

Reply:
https://github.com/HathorNetwork/hathor-wallet-mobile/pull/498